orbitx/Vpython.py

```diff
@@ -0,0 +1,31 @@
+import vpython
+import signal
+
+vpython_object = vpython
+container = [vpython_object]
+closed = False
+
+def vpython_version_check() -> None :
+    print("version check is executed")
```

Comment:
is there a reason we use a `print` call instead of a `log.info` call?

Reply:
It was for testing to see how many times version check is executed. It should be deleted.